kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -259,15 +288,28 @@
       search(query) {
         if (!query) {
           this.setSearchResult({});
+          this.resultZim = [];
           return;
         }
 
         this.progress = 0;
         kinds.forEach(k => {
           searchChannels(this.$store, query, k).then(() => {
-            this.progress += 100 / kinds.length;
+            this.progress += 100 / this.totalSearchSections;
           });
         });
+
+        this.resultZim = [];
+        for (const k in ZimSearchChannels) {
+          wikiChannelSearch(ZimSearchChannels[k], query)
+            .then(r => {
+              this.resultZim.push([k, r]);
```

Comment:
Please make `k` and `r` be readable, understandable names. I see that `ZimSearchChannels` is actually groups of channels, multiple WikiHow channels and the Wikipedia with only one ID in the group.